### PR TITLE
misc fix spell errors in osd/OSD and doc

### PR DESCRIPTION
**Pull request by Songweibin**
Signed-off-by: songweibin <song.weibin@zte.com.cn>

### CODE REVIEW

File: src/osd/OSD.cc [hunk 1]
```diff
@@ -9843,7 +9843,7 @@ int OSD::init_op_flags(OpRequestRef& op)
     if ((iter->op.op == CEPH_OSD_OP_WATCH &&
 	 iter->op.watch.op == CEPH_OSD_WATCH_OP_PING)) {
       /* This a bit odd.  PING isn't actually a write.  It can't
-       * result in an update to the object_info.  PINGs also aren'ty
+       * result in an update to the object_info.  PINGs also aren't
        * resent, so there's no reason to write out a log entry
```
Comment (joscollin) [inline src/osd/OSD.cc]: I think the word `resent` is wrong here. Please check.

Comment (xiexingguo) [inline src/osd/OSD.cc]: it is the past participle of "resend"

Comment (joscollin) [inline src/osd/OSD.cc]: Oh!. I thought it is:
```
resent
rɪˈzɛnt/
verb
verb: resent; 3rd person present: resents; past tense: resented; past participle: resented; gerund or present participle: resenting

    feel bitterness or indignation at (a circumstance, action, or person).
    "she resented the fact that I had children"
```

Comment (Songweibin) [inline src/osd/OSD.cc]: Added a period at the end. Thanks.

**Pull request merged**